chassis/core/src/main/java/com/griddynamics/jagger/master/Master.java

```diff
@@ -277,6 +277,9 @@ public void run() {
                     listener.onSessionExecuted(sessionId, metaDataStorage.getComment());
                 }
             }
```

Comment:
Comments should go below target lines not above.
It is more correct behaviour to stop agent right away after test execution and only then go on to pdf generation